pages/common/docker.md

```diff
@@ -23,7 +23,7 @@
 
 `docker pull {{image}}`
 
-- Open a shell inside of an already running container:
+- Open a shell inside an already running container:
```

Comment:
?
```suggestion
- Open a shell inside an already-running container:
```

Reply:
Not 100% sure about this one, but I can find examples of "already-running" on Wikipedia when used in a similar context. 

> This results in a delay to the handling of the connection (in comparison to a connection handled by an **already-running** process). 
> \- https://en.wikipedia.org/wiki/Super-server#Performance

However, most of the time it's not done like this. (The example above was only because I specifically searched for it.)

It looks like "already-running" isn't on Merriam-Webster, Lexico, or Collins dictionary either.

To be honest, I feel this should just say "a running container" anyway. The "already" doesn't add anything of value.

```suggestion
- Open a shell inside a running container:
```

Thoughts?

Reply:
Like it. 👍🏻